### ISSUE TITLE
Confidence

### ISSUE THREAD

**Issue by zallia2** [Microsoft/Cognitive-Face-Android#37]
Hi what is the general rule of thumb for the confidence in Microsoft Cognitive Services? Do you have any graph or anything available which states if it is less than 50% don't trust the results and if it is above 50% do trust the results. Like also if it is 90%+ it is extremely accurate results etc.

**Comment by lebronJ** [comment]
As you can see from the Face API reference:

1. [verify](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f3039523a)
"A number indicates the similarity confidence of whether two faces belong to the same person, or whether the face belongs to the person. By default, isIdentical is set to True if similarity confidence is greater than or equal to 0.5. This is useful for advanced users to override "isIdentical" and fine-tune the result on their own data."

2. [Identify](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395239)
"Customized identification confidence threshold, in the range of [0, 1]. Advanced user can tweak this value to override default internal threshold for better precision on their scenario data. Note there is no guarantee of this threshold value working on other data and after algorithm updates."

3. [FindSimilar](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395237)
Similarity confidence of the candidate face. The higher confidence, the more similar. Range between [0,1].

Basically, 
1. The higher confidence, the more similar.
2. 50% is a default threshold for verify & identify, however this can be changed in later upgrades, and it is suggested for advanced users to fine-tune the threshold on their own data.
